src/ScottPlot5/ScottPlot5/Plot.cs

```diff
@@ -376,6 +379,20 @@ public string SaveWebp(string path, int width, int height, int quality = 85)
         return Path.GetFullPath(path);
     }
 
+    public string Save(string path, int width, int height, int quality = 85)
```

Comment:
I actually omitted this function intentionally, but I'm interested in your thoughts. 

My line of thinking is that some file formats accept `int quality` and others may accept `bool transparency`, and I didn't want to force those arguments for file types that don't support them, hence separate SaveBmp, SavePng, SaveJpg, etc.

Reply:
... I also don't love the idea of users evaluating what extensions are supported by trial and error and seeking runtime exceptions 😅 Having dedicated save methods for specific file formats solves that issue.

Reply:
I think it's ok to pass in a `quality` that promptly gets ignored, I think that's what `GetImageBytes` and `SKImage.Encode` do anyways.

Perhaps a good middleground is a `public static ImageFormat FromFilePath(string fpath)` which returns an enum with fields like `ImageFormat.Png`, `ImageFormat.Jpeg`, etc etc, just so that we don't have to handle the extension conversions everywhere.

Reply:
Good point!

https://github.com/ScottPlot/ScottPlot/blob/282a94a386ed59e252d877f55defa02990548034/src/ScottPlot5/ScottPlot5/ImageFormat.cs#L1-L9

I wish there were a way we could do something like 

```cs
ImageFormat fmt = ImageFormat.FromFilename("test.png");
```

Maybe there should be an `ImageFormats` class that has a `GetImageFormat(string filename)` function? Perhaps the various logic that saves `SKSurface` objects as files could be moved there and out of the `Plot` module

Reply:
> I wish there were a way we could do something like

Enums support extension methods

Reply:
lol, duh, I don't know why I got confused by that 🤪

I was thinking it would have to be a `string` extension method 🤦 

Yes, enum extension method, great option!

Reply:
I brainfarted, you can't create an enum through its own extension method since extension methods require an instance. I just created an `ImageFormatHelpers` class with static methods `FromFileExtension` and `FromFilePath`